nimsuggest/nimsuggest.nim

```diff
@@ -114,6 +114,12 @@ proc writelnToChannel(line: string) =
 proc sugResultHook(s: Suggest) =
   results.send(s)
 
+proc log*(s: string) =
```

Comment:
```suggestion
proc log(s: string) =
  ## Appends `s` as new line to the `nimsuggest.log` file in the user's home
  ## directory. 
```
Since it's only used in this module, the procedure doesn't need to be exported.